dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -1050,10 +1041,6 @@ BEGIN
     IDS_ADVANCED_DISPLAY_ADMINTOOLS "Se afişează Instrumente de administrare"
     IDS_ADVANCED_SMALL_START_MENU "Se afişează Pictograme mici în Meniul Start"
 
-    IDS_VISEFFECT_DRAGFULL "Show window contents while dragging"
-    IDS_VISEFFECT_FONTSMOOTHING "Smooth edges of screen fonts"
-    IDS_VISEFFECT_LISTVIEWSHADOW "Use drop shadows for icon labels on the desktop"
```

Comment:
Please restore these three (and translate them if you wish).

Reply:
Done!